jiwer/process.py

```diff
@@ -189,36 +193,45 @@ def process_words(
     alignments = []
 
     for reference_sentence, hypothesis_sentence in zip(ref_as_chars, hyp_as_chars):
```

Comment:
nit: Would you be able to change _as_chars to _as_ints?

Reply:
Sure! Done in https://github.com/jitsi/jiwer/pull/100/commits/0fe59c0d2c24f1588239acdc71a216e322c85a45